.github/workflows/build-extensions.yml

```diff
@@ -116,6 +116,7 @@ jobs:
           bash utils/wasi-nn/install-openvino.sh
           bash utils/wasi-nn/install-pytorch.sh
           bash utils/wasi-nn/install-neuralspeed.sh
+          bash utils/wasi-nn/install-onnxruntime-ubuntu.sh
```

Comment:
I think we don't need `-ubuntu` suffix here? @hydai how do you think?

Reply:
We should remove the suffix because this should also be used in the manylinux(CentOS) distributions.